service/metrics/metrics.go

```diff
@@ -169,14 +170,14 @@ func (m *metricsImpl) ObserveMessageSize(messageType MessageType, size int) {
 func (m *metricsImpl) ObservePriceForTicker(ticker slinkytypes.CurrencyPair, price float64) {
 	m.prices.With(prometheus.Labels{
 		ChainIDLabel: m.chainID,
-		TickerLabel:  ticker.String(),
+		TickerLabel:  strings.ToLower(ticker.String()),
```

Comment:
I feel like we should be doing the opposite here.

On chain the tickers are BTC/USD.  and also internally in the sidecar we format them like that.

I feel like the metrics should be using the canonical way we represent tickers

Reply:
side car lower cases everything

Reply:
along with metrics

Reply:
we're going to have to make many more changes to get everything to upper-case, i prefer just sticking with this unless the vals are very opinionated

Reply:
I would still prefer to upper case everything because that unifies everything.

Say I'm looking at metrics and see the price for `btc/usd`, then i run an on-chain query for it using "btc/usd".  It will appear as thought the price isnt there.

Reply:
If this seems unnecessary or too much work im fine with not doing this tho.  its not too big of a deal or vals don't care

Reply:
yea i'd prefer to discuss this internally before we go with it. down for either, the only hesitation is that it'll require more changes.

Reply:
probably best for a separate task.